.ci/system-tests/run-system-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Script to run the System Tests for HIRS
```

Comment:
If this file exists, I think we can delete the original `run-integration-tests.sh` file.